packages/scraper-tools/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
also we dont need licenses in packages, there is on in the root of the repo that gets copied to each package during build (thats the `copy.ts` script)